Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -776,7 +776,6 @@ def http_request(
         token = self.get_access_token(resource=resource, scope=scope)
         default_headers = {
             'Authorization': f'Bearer {token}',
-            'Content-Type': 'application/json',
```

Comment:
The documentation is specifying "multipart/form-data" but we can't specify that as a header because in the http standard, form-data need a boundary. Eg: `Content-Type:multipart/form-data; boundary=---------------------------293582696224464`. So setting it to only "multipart/form-data" will not works due to the missing boundary.
You can see a more detailed explanation here: https://stackoverflow.com/a/12385661
So we can't override the content-type header with the `headers` parameter and removing the default `application/json` is required.